dash/dash_table/__init__.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import print_function
```

Comment:
Ah, there will be a little merge conflict with https://github.com/plotly/dash/pull/1734 where we remove all these build artifacts... (`dash/dash_table`, `dash/dcc`, `dash/html`)